interface/VideoEncoderDefs.h

```diff
@@ -189,10 +189,14 @@ typedef struct VideoRateControlParams {
     uint32_t targetPercentage;
     uint32_t disableFrameSkip;
     uint32_t disableBitsStuffing;
+    uint32_t diffIPQP;// I/P frame QP difference
+    uint32_t diffPBQP;// P/B frame QP difference,
```

Comment:
Hi, LiZhong, diffPBQP is assigned to qp_diff_pb in libva too, so the name diffPBQP may be better.